js/main.js

```diff
@@ -52,6 +51,7 @@
     var apiUrl      = baseAPIsUrl + "2";
     var assetsUrl   = "https://classroom-assets.frontrowed.com";
     var imgixUrl    = "https://frontrow-image-assets.imgix.net";
+    var oldImgixUrl    = "https://frontrow-image-assets.imgix.net";
```

Comment:
Is this just future proofing? This is the same as the previous URL.

Reply:
Lollllll, just me being dumb.